server/routes/api/auth.js

```diff
@@ -22,4 +25,52 @@ router.post('/login', (req, res) => {
     });
 });
 
+router.get('/logout', (req, res) => {
+  if (req.session.userId) {
+    req.session.destroy(err => {
+      err ? res.status(500).json(err) : res.status(200)
+    })
+  }
+  res.end()
```

Comment:
Not sure why this logout code block shows in the diff, but since it does, I noticed something. Let's go ahead and toss `res.end()` in an else statement because `res.session.destroy()` seems to be asynchronous. We wouldn't want `res.send()` to be called before it's finished.